config/environments/production.rb

```diff
@@ -100,4 +100,11 @@
     ExceptionNotification::Rack,
     slack: { webhook_url: 'https://hooks.slack.com/services/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' }
   )
+
+  config.secret_path = '/run/secrets'
+    Raven.configure do |config|
+    config.dsn = "https://#{OperationCode.fetch_secret_with(name: :sentry_credentials)}@sentry.io/147247"
+    config.environments = ['production']
+    config.release = '0e4fdef81448dcfa0e16ecc4433ff3997aa53572'
```

Comment:
SENTRY_RELEASE sets the application release version for the client to use, this is usually a Git SHA hash - it's not well explained from that page :/

https://github.com/getsentry/sentry/issues/3081